### PR TITLE
code/dte: add XML schemas of "factura electrónica"

### PR DESCRIPTION
**Pull request by yaselc**
Official XML schemas of DTE (Documento Tributario Electrónico)

Includes a README file by @glarrain

All the files have been preserved as they were; schemas are in their original text encoding (ISO-8859-1) and have not been modified in the slightest way.

Sources (2021-03-25):
http://www.sii.cl/factura_electronica/schema_dte.zip

Ref: #11 
Replaces #12

### CODE REVIEW

File: src/code/dte/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# SII "Documento Tributario Electrónico (DTE)"
+
+
+## XML schemas
+
+- `DTE_v10.xsd`: "XSD principal y que incluye a los 3" otros XSD.
+  - Version 2011-05-30(latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
```
Comment (jtrh) [inline src/code/dte/README.md]:
```suggestion
  - Version 2011-05-30 (latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
```

File: src/code/dte/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## XML schemas
+
+- `DTE_v10.xsd`: "XSD principal y que incluye a los 3" otros XSD.
+  - Version 2011-05-30(latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
+    - MD5: `e72ee34d798224f31a5127edda712bbf`.
+    - SHA256: `5418f69b9ee64238b337e4d91fd255c567bc86353b15bb192812459b304067e8`
+
+- `EnvioDTE_v10.xsd`: "descripción de documentos"
+  - Version 2011-05-30(latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
```
Comment (jtrh) [inline src/code/dte/README.md]:
```suggestion
  - Version 2011-05-30 (latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
```

File: src/code/dte/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    - MD5: `6ed5ceeb7508df3c9d6e99f955751a94`.
+    - SHA256: `e86aae6ab21cbce8e93ddcf2df4f8b80cac42bab3c8da845d0756917e866aa2a`
+
+- `SiiTypes_v10.xsd`: "descripción de tipos de datos"
+  - Version 2011-05-30(latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
```
Comment (jtrh) [inline src/code/dte/README.md]:
```suggestion
  - Version 2011-05-30 (latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
```

File: src/code/dte/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    - MD5: `b6a63aa427e3d528e46a7d94ccbdcb32`.
+    - SHA256: `b6ba16b1698cae563b7aabde476f080b5f723769c6c2bb0c49b95e14da29b4dd`
+	
+- `xmldsignature_v10.xsd`: "descripción de la firma electrónica"
+  - Version 2011-05-30(latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
```
Comment (jtrh) [inline src/code/dte/README.md]:
```suggestion
  - Version 2011-05-30 (latest): [2011-05-30-schema_dte/schema_dte](2011-05-30-schema_dte/schema_dte)
```

File: src/code/dte/2011-05-30-schema_dte/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+## Contents
+
+
+### Documentos Tributarios Electrónicos
+
+"Schema XML de Documentos Tributarios Electrónicos",
+[2011-05-30](2011-05-30-schema_dte.zip)
+
+- Fuente: [SII](http://www.sii.cl)
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/README.md]:
```suggestion
- Fuente: [SII](https://www.sii.cl)
```

File: src/code/dte/2011-05-30-schema_dte/README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  / [Factura electrónica](http://www.sii.cl/servicios_online/1039-.html)
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/README.md]:
```suggestion
  / [Factura electrónica](https://www.sii.cl/servicios_online/1039-.html)
```

File: src/code/dte/2011-05-30-schema_dte/README.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  / [FORMATO XML DE DOCUMENTOS ELECTRÓNICOS](http://www.sii.cl/factura_electronica/formato_xml.htm)
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/README.md]:
```suggestion
  / [FORMATO XML DE DOCUMENTOS ELECTRÓNICOS](https://www.sii.cl/factura_electronica/formato_xml.htm)
```

File: src/code/dte/2011-05-30-schema_dte/README.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  as
+  "[Bajar schema XML de Documentos Tributarios Electrónicos](http://www.sii.cl/factura_electronica/schema_dte.zip) (Incluye Documentos de exportación)"
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/README.md]:
```suggestion
  "[Bajar schema XML de Documentos Tributarios Electrónicos](https://www.sii.cl/factura_electronica/schema_dte.zip) (Incluye Documentos de exportación)"
```

File: src/code/dte/README.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# SII "Documento Tributario Electrónico (DTE)"
```
Comment (jtrh) [inline src/code/dte/README.md]: Replace Windows line terminators (`CR LF`, `\r\n`) with Unix line terminators (`LF`, `\n`).

```
src/code/dte/README.md:  UTF-8 Unicode text, with CRLF line terminators
```

File: src/code/dte/2011-05-30-schema_dte/schema_dte/README.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
+# SII "Documento Tributario Electrónico (DTE)" / XML schemas
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/schema_dte/README.md]: Replace Windows line terminators (`CR LF`, `\r\n`) with Unix line terminators (`LF`, `\n`).

```
src/code/dte/2011-05-30-schema_dte/schema_dte/README.md: UTF-8 Unicode text, with CRLF line terminators
```

File: src/code/dte/2011-05-30-schema_dte/schema_dte/README.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+
+
+## Source
+
+
+### Original & Official
+
+[schema_dte.zip](../README.md) (2011-05-30),
+referenced from official webpage
+[SII](http://www.sii.cl)
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/schema_dte/README.md]:
```suggestion
[SII](https://www.sii.cl)
```

File: src/code/dte/2011-05-30-schema_dte/schema_dte/README.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+/ [Factura electrónica](http://www.sii.cl/servicios_online/1039-.html)
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/schema_dte/README.md]:
```suggestion
/ [Factura electrónica](https://www.sii.cl/servicios_online/1039-.html)
```

File: src/code/dte/2011-05-30-schema_dte/schema_dte/README.md [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+/ [FORMATO XML DE DOCUMENTOS ELECTRÓNICOS](http://www.sii.cl/factura_electronica/formato_xml.htm)
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/schema_dte/README.md]:
```suggestion
/ [FORMATO XML DE DOCUMENTOS ELECTRÓNICOS](https://www.sii.cl/factura_electronica/formato_xml.htm)
```

File: src/code/dte/2011-05-30-schema_dte/schema_dte/README.md [hunk 14]
```diff
@@ -0,0 +1,118 @@
...
+as
+"[Bajar schema XML de Documentos Tributarios Electrónicos](http://www.sii.cl/factura_electronica/schema_dte.zip) (Incluye Documentos de exportación)"
```
Comment (jtrh) [inline src/code/dte/2011-05-30-schema_dte/schema_dte/README.md]:
```suggestion
"[Bajar schema XML de Documentos Tributarios Electrónicos](https://www.sii.cl/factura_electronica/schema_dte.zip) (Incluye Documentos de exportación)"
```

**Review by jtrh — CHANGES_REQUESTED** [review]

**Review by glarrain — APPROVED** [review]

**Review by jtrh — APPROVED** [review]

**Pull request merged**